pallets/msa/src/tests.rs

```diff
@@ -96,6 +96,31 @@ fn it_throws_error_when_key_verification_fails() {
 	});
 }
 
+#[test]
+fn it_throws_error_when_key_verification_fails_using_mock_create_account() {
```

Comment:
Looking good.

Reply:
Should I replace my tests with the tests that use `Msa::create_account`?

Reply:
Not 100% sure what you are asking.

Reply:
The issue was that there was repeated code with the two key pairs. In mock.rs, `create_account` creates an MSA id, and this function is used in my tests. These tests could replace the tests that use `Msa::create_acount` right?

Reply:
Assuming that the tests are otherwise the same, yes they could replace the old ones.  Please delete the `_using_mock_create_account` part of the test name though, otherwise the test name almost sounds like it's not a "real" test, when in fact it does call Msa::create_account.